### PR TITLE
[Docs] Minor typos

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
Also removed reference to "of each accumulator" as exgauss doesn't have multiple accumualtors

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
By the way, what do you think about making the Wald & Exgauss compatible with a specification of type:

`ExGaussian([0.8], 0.5, 0.3)`

i.e., accepting a vector of length 1 for drift/sigma? I think it might add the (small) benefit of being able to have a consistent syntax accross models in Turing (i.e., using filldist and vectorized operations) with other models. 
(I'm obviously biased given my current usecase of fitting different models to the same no-choice data)

File: docs/src/lca.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Leaky Competing Accumulator
 
-The Leaky Competing Accumulator (LCA; Usher & McClelland, 2001) is a sequential sampling model in which evidence for options races independently. The LBA makes an additional simplification that evidence accumulates in a linear and ballistic fashion, meaning there is no intra-trial noise. Instead, evidence accumulates deterministically and linearly until it hits the threshold.
+The Leaky Competing Accumulator (LCA; Usher & McClelland, 2001) is a sequential sampling model in which evidence for options races independently. the LCA is similar to the Linear Ballistic Accumulator (LBA), but additionally assumes an intra-trial noise in the form of evidence accumulation leakage (in contrast, the The LBA assumes that evidence accumulates in a ballistic fashion, i.e., linearly and deterministically until it hits the threshold).
```
Comment (itsdfish) [inline docs/src/lca.md:3]: The needs to be capitalized.

Comment (itsdfish) [inline docs/src/lca.md:3]: "the The" in the parentheses

Comment (itsdfish) [reply docs/src/lca.md:3]: Noise and leakage are separate concepts. I would say something like:

"intra-trial noise and leakage (in contrast, the The LBA assumes that evidence accumulates in a ballistic fashion, i.e., linearly and deterministically until it hits the threshold)

**Comment by itsdfish** [comment]
Thanks for the fixes to the docs. There were a few minor issues in the changes. 


I'll have to think about it some more, but I don't think making the drift rate a vector is a good idea. Using a scalar communicates to the user that there is only one drift rate, and prevents accidental use of multiple drift rates.

**Comment by DominiqueMakowski** [comment]
> Using a scalar communicates to the user that there is only one drift rate

Fair (though one could also throw an informative error "only one accumulator is possible for this model")

**Comment by DominiqueMakowski** [comment]
Other minor consistency thing:

LBA has the following order: drift, A, k, τ
But RDM has: drift, k, A, τ 

Does it make sense to standardize when possible?

**Comment by itsdfish** [comment]
Yes. Standardizing the order is on my agenda. It became apparent I should do that when you opened the issue about the LBA doc strings the other day. Thanks for the suggestion.

**Pull request merged**